test_framework/scripts/longhorn-setup.sh

```diff
@@ -351,6 +351,17 @@ create_aws_secret(){
 }
 
 
+longhornctl_check(){
+  curl -L https://github.com/longhorn/cli/releases/download/v1.7.0-rc2/longhornctl-linux-amd64 -o longhornctl
```

Comment:
For `arm64`, we should download https://github.com/longhorn/cli/releases/download/v1.7.0-rc2/longhornctl-linux-arm64 instead of `longhornctl-linux-amd64`, right?

Reply:
No. it’s run in a Jenkins agent docker container instead of the nodes you created.